src/collections/posts/hidden-features-of-beatsaver.md

```diff
@@ -0,0 +1,91 @@
+
+# Excluding Tags
+
+By opening the filter dropdown you gain the ability to filter for tags. For example, if you are into Tech maps, you can search for Tech maps specifically.
+
+![](/uploads/posts/uncovering-beatsaber-features-1.png)
+
+At first glance, it appears to only allow for positive filtering; meaning you can only declare "I want to only be shown maps that have the XYZ tag".
+
+But what if you are having a lazy day and want to avoid Fitbeat maps? Just press the `[Alt]` button while clicking the tag that you want to exclude. You will find that the tag turns red, and maps with such a tag will no longer appear in your search!
+
+![](/uploads/posts/uncovering-beatsaber-features-2.png)
+
+# Selecting Multiple Tags
+
+When playing with the tag filters, you will notice that you can only select one of the blue tags and one of the green tags. Selecting another of the same category will result in the previous selection being deleted. However, it is possible to select multiple tags by pressing `[Shift]` while clicking the additional tags.
+
+![](/uploads/posts/uncovering-beatsaber-features-3.png)
+
+The results from the image above will now show you maps that either have the "Fitness" or "Tech" or both tags. If you were trying to find maps that are techy Fitbeat maps you cannot find them this way though.
+
+However, you can combine it with the tag exclusion feature and find your techy Fitbeat maps by excluding every tag that is not "Tech" and "Fitness" by pressing `[Shift]` and `[Alt]` together while clicking the tags.
+
+![](/uploads/posts/uncovering-beatsaber-features-4.png)
+
+This works because multiple negative filters will - other than the positive filters - result in being interpreted as "Neither X, Y, nor Z or any combination of those".
+
+# Multi-Adding to Bookmarks
+
+On playlists you have a useful "Multi-Add" button that you can use to add multiple maps by their key or hash.
+
+![](/uploads/posts/uncovering-beatsaber-features-5.png)
+
+When it comes to your bookmarks, you will not find buttons like that.
+
+![](/uploads/posts/uncovering-beatsaber-features-6.png)
+
+Do not let that fool you! Just add a `/add` to the end of your bookmark URL and you can still multi-add! For example, `https://beatsaver.com/playlists/97886/add` (this link will not work for you though).
+
+You can also add `/edit` to the URL, but this will not do a lot for you since the API will not allow editing the playlist.
+
+# "key:" keyword in search
+
+If you know the key of a map, you can navigate to the map page directly by typing
+
+`key:{your map key}`
+
+in the search bar and then clicking "Search".
+
+![](/uploads/posts/uncovering-beatsaber-features-7.png)
+
+# No Double Upload
+
+When you try to upload a map that has already been uploaded it will not allow you to do so and tell you that the Map has already been uploaded. This is checked via the map hash. The upload will also generally verify, that your map file is valid and will work in-game.
```

Comment:
When you try to upload a map that has already been uploaded, it will not allow you to do so and will tell you that the map has already been uploaded. This is checked via the map hash. The upload will also generally verify that your map file is valid and will work in-game.